app/src/main/java/org/eyeseetea/malariacare/presentation/presenters/DownloadedMediaPresenter.java

```diff
@@ -0,0 +1,64 @@
+package org.eyeseetea.malariacare.presentation.presenters;
+
+import org.eyeseetea.malariacare.domain.entity.Media;
+import org.eyeseetea.malariacare.domain.usecase.observation.GetMediasUseCase;
+import org.eyeseetea.malariacare.factories.MediaFactory;
+
+import java.util.List;
+
+public class DownloadedMediaPresenter {
+
+    View mView;
+
+
+    public void attachView(final View view) {
+        mView = view;
+        loadMedia();
+    }
+
+    private void loadMedia() {
+        MediaFactory mediaFactory = new MediaFactory();
```

Comment:
A presenter should not invoke factory to retrieve use case, the use case should be passed via constructor